dox/books/reference_manual/DoxyRefman

```diff
@@ -790,10 +790,7 @@ RECURSIVE              = NO
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = ./test_* \
-                         ~/.. ./.. .. ../.. ../../.. \
-                         /home/ /home/j/ /home/j/dev \
-                         dox/books/reference_manual/reference.dox
+EXCLUDE                = dox/books/reference_manual/reference.dox
```

Comment:
I am confused.  Why remove refrence.dox?  Or is it that reference.dox is for html while main.dox is for PDFs?

If that is the case, consider a naming convention such as main.dox (which is a \mainpage) and main_html.dox (which is a \page).

If you are permitted by doxygen, consider main_pdf.dox instead of main.dox.